DB-SCHEMA.md

```diff
@@ -0,0 +1,33 @@
+# Database Schema Docs
+
+## Countries Table 
+
+| Column Name | Data Types | Description   |
+|------------------------------------------|
+| id          | string     | Unique id for the country          
+| name        | string     | Name of country
+
+## Players Table
+
+| Column Name | Data Types | Description   |
+|------------------------------------------|
+| id          | string     | Unique id for the country          
+| name        | string     | Name of country
```

Comment:
Do we need country name if we have `country_id`? If so, we should probably use `country_name` instead of just `name`.